File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        i=0
        j=0
        unique=1
        while i<len(nums):
            if nums[j]!=nums[i]:
                nums[j+1]=nums[i]
                j+=1
                unique+=1
            i+=1
        return unique

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {

        int start=0;
        int unique=1;
        for(int i=1;i<nums.length;i++){
            if(nums[start]==nums[i]){
                continue;
            }else{
                start++;
                nums[start]=nums[i];
                unique++;
            }
        }
        return unique;
    }
}